api/src/form_schema/field_info.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from dataclasses import dataclass
+from typing import Any
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class FieldInfo:
+    id: str
+    required: bool
+    data_type: str
+    type: str
+    label: str
+    min_value: int | None = None
+    max_value: int | None = None
+    list_of_values: list[str] | None = None
+    help_tip: str | None = None
+    min_occurrences: int | None = None
+    is_nullable: bool = False
+
+    @staticmethod
+    def parse_numeric(value: str | None) -> int | None:
+        """
+        Convert a string to a numeric value (int or float).
+
+        Args:
+            value: String value to convert to numeric
+
+        Returns:
+            Integer or float value, or None if input is empty/None
+        """
+        if value is None or value.strip() == "":
+            return None
+
+        # Remove any currency symbols, commas, and whitespace
+        cleaned_value: str = value.strip().replace("$", "").replace(",", "").replace(" ", "")
+
+        try:
+            # Try converting to int first
+            num_value = int(cleaned_value)
+            return num_value
+        except ValueError:
+            # Log the error or handle it as needed
+            logger.warning(f"Warning: Could not convert '{value}' to a numeric value")
+            return None
```

Comment:
When running locally, I get a lot of warnings for `Warning: Could not convert '1.0' to a numeric value` - seems like the numbers are stored as floats but we aren't converting them to ints as we'd expect.

Apparently `int("1.0")` isn't valid, so we'll have to do something like `int(Decimal("1.0"))`